networkd/network_server.ml

```diff
@@ -68,13 +68,13 @@ let on_timer () =
 
 let reopen_logs _ () = true
```

Comment:
Let's remove `_` here as well

Reply:
I'd rather delete the function as it is out of use.